code/src/lidar.cpp

```diff
@@ -1,89 +0,0 @@
-#include <lidar.hpp>
```

Comment:
Why is this file deleted?

Reply:
lidar, shouldn't be here

Reply:
I returned the file and corrected the outdated usart.